### PR TITLE
Fixed the toast message for deleting an org.

### LINKED ISSUE #1108: Toast Messages not working while Deleting Organization.
**Describe the bug**
When we delete an organization or a sample organization, no toast message of success is shown.

**To Reproduce**
Steps to reproduce behaviour:

1. Create a sample organization and an organization.
2. Go to the settings of these organizations.
3. Click on the Delete Organization button.
4. Now, you can see that no toast message is shown for both the sample and the created organization.

**Expected behaviour**
There should be a Toast message of success whenever we Delete an organization.

**Actual behaviour**
No Toast message is shown whenever an organization is deleted.

**Screenshots**
Before fixing :
https://github.com/PalisadoesFoundation/talawa-admin/assets/110725065/a6ce0fa6-9e3c-4841-865c-066d590b66b4

**Additional details**
I want to work on this issue.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by AmitSharma512**
**What kind of change does this PR introduce?**

This PR solves the issue of the toast message not being displayed while deleting the organization.

**Issue Number:**

Fixes #1108 

**Snapshots/Videos:**

https://github.com/PalisadoesFoundation/talawa-admin/assets/110725065/07138238-720d-4d4e-b31c-9b190ccb8a1a

**Summary**

Previously, When we delete an organization or a sample organization, no toast message of success was shown, and instead of using React Routing, window.location.replace was used. This PR solves all the above issues.

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
Please fix the failing tests.

File: src/components/DeleteOrg/DeleteOrg.tsx [hunk 1]
```diff
@@ -29,24 +30,26 @@ function deleteOrg(): JSX.Element {
...
-        .catch((error) => {
-          toast.error(error.message);
-        });
-      window.location.replace('/orglist');
+      try {
+        await removeSampleOrganization();
+        toast.success('Successfully deleted sample Organization');
+        history.push('/orglist');
+      } catch (error: any) {
+        errorHandler(t, error);
```
Comment (noman2002) [inline src/components/DeleteOrg/DeleteOrg.tsx:42]: Write tests for these lines.

File: src/components/DeleteOrg/DeleteOrg.tsx [hunk 2]
```diff
@@ -29,24 +30,26 @@ function deleteOrg(): JSX.Element {
...
     } else {
       try {
         await del({
           variables: {
             id: currentUrl,
           },
         });
-        window.location.replace('/orglist');
+        toast.success('Successfully deleted Organization');
+        history.push('/orglist');
```
Comment (noman2002) [inline src/components/DeleteOrg/DeleteOrg.tsx:52]: These too.

**Review by noman2002 — CHANGES_REQUESTED** [review]

**Comment by AmitSharma512** [comment]
@noman2002 I tried to write the tests, but every time, I got an unexpected error that I was not able to solve.
According to my knowledge, my basics are correct, but then also tests are not passing.

**Comment by noman2002** [comment]
@AmitSharma512 Ask in the slack channel, someone will help you out.

**Comment by noman2002** [comment]
@AmitSharma512 Any update on this one ?

**Comment by AmitSharma512** [comment]
@noman2002 actually I was out of station for the last 3 days, from tomorrow I will be back to work.

**Comment by palisadoes** [comment]
The test code coverage is dropping significantly with this PR. Please get to at least 95% coverage for all your edited files. Though there may be issues to increase the coverage for the file, the aim is for them to reach 100% coverage

**Comment by palisadoes** [comment]
Is this still being worked on?

**Comment by AmitSharma512** [comment]
@palisadoes yes

**Comment by noman2002** [comment]
@AmitSharma512 Its been more than 3 weeks since this PR is open. Please update on this or I have to close it.

**Comment by palisadoes** [comment]
Closing and unassigning. It's been too long and the task is simple

**Pull request closed without merging**